user_simulation/simulate_user.py

```diff
@@ -1,21 +1,21 @@
 from selenium import webdriver
-from selenium.webdriver.firefox.service import Service
 from selenium.webdriver.common.by import By
 from selenium.webdriver.common.keys import Keys
 from selenium.webdriver.common.action_chains import ActionChains
+REPLACE_ME_1
 import time
 import threading
-import shutil
-import tempfile
 
 
 
 class User:
 	
 	def __init__(self, credentials):
 		self.credentials = credentials
-		service = Service("geckodriver")
-		self.driver = webdriver.Firefox(service=service)
+
+		REPLACE_ME_2
+		REPLACE_ME_3
+
```

Comment:
Could we potentially set an environment variable to determine what browser is used? I think that could be a cleaner solution then replacing python code at run time

Reply:
Yeah I wan't quite sure what the best way would be. Right now, these variables are replaced permanently once the install script is run. This way, we can ask the user what browser they want to use (in the install) only once, and then everything will be set up. I wanted to ask the user, since it's possible for someone to have both browsers. It's definitely a little messy though, so if you think it would be better to try an environment variable instead I can change it.